app-android/src/main/java/com/ustadmobile/port/android/view/Login2Fragment.kt

```diff
@@ -119,5 +142,94 @@ class Login2Fragment : UstadBaseFragment(), Login2View {
         mPresenter = null
         mBinding = null
     }
+}
+
+@Composable
+private fun LoginScreen(
+    uiState: LoginUiState = LoginUiState(),
+    onClickLogin: () -> Unit = {},
+    onClickCreateAccount: () -> Unit = {},
+    onClickConnectAsGuest: () -> Unit = {},
+    onUsernameValueChange: (String) -> Unit = {},
+    onPasswordValueChange: (String) -> Unit = {},
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally
+    )  {
+
```

Comment:
Login Intent message (if not null) should be displayed here above the text fields.